### ISSUE TITLE
protocol control parameters support for crypto

### ISSUE THREAD

**Issue by vitalz** [agrestio/agrest#371]
Whenever a Cayenne entity is crypto then protocol [_control parameters_](https://agrest.io/docs/protocol.html#control-parameters) ain't applied. 
For now, at least these params are mostly required for crypto apps:
- cayenneExp
- sort
- dir
- start
- limit
- page

Good options are:
- mapBy
- include / exclude

**Comment by andrus** [comment]
I think this is more nuanced that that. Just from memory:

* All the parameters above should work if they don't reference any encrypted properties of an entity that has both encrypted and unencryptyed properties. 
* Even with encrypted properties I think many parameters will still work. Namely `include`, `exclude`, `start`, `limit` should not be affected.
* `cayenneExp` on the other hand will not work if it references encrypted properties. Same for `sort`

**Comment by vitalz** [comment]
@andrus Yes, you're correct _start_, _limit_ are not affected. But these ones should work with cayenneExp on encrypted properties.
